kt/godot-library/src/main/kotlin/godot/extensions/NodeExt.kt

```diff
@@ -0,0 +1,10 @@
+package godot.extensions
+
+import godot.Node
+import godot.core.NodePath
+
+@Suppress("NOTHING_TO_INLINE", "UNCHECKED_CAST")
+inline fun <T : Node> Node.node(path: String) = getNode(NodePath(path)) as T?
+
+@Suppress("NOTHING_TO_INLINE", "UNCHECKED_CAST")
+inline fun <T : Node> Node.node(nodePath: NodePath) = getNode(nodePath) as T?
```

Comment:
Why is the rename necessary? It works for me if they just have a different package path.

Reply:
It works for getNode(String) but not for getNode(NodePath) in my case.

Reply:
![1.png](https://user-images.githubusercontent.com/22662033/115974236-937d4700-a55b-11eb-8497-c67798e2ecf6.png)
Hmm it does for me

Reply:
it's suggested but when I select it, it still doesn't work, the import is not made so I just get a red line.

Reply:
might be good to test out latest kotlin versions and enabling JVM IR - from what I understand the code still compiles but the only IDE marks it as red?